templates/patient_extension.rb.erb

```diff
@@ -23,6 +23,17 @@
     dataElements.where(qrdaOid: qrda_oid) || []
   end
 
+  # Shift all data element fields that deal with dates by the given value.
+  # Given value should be in seconds. Positive values shift forward, negative
+  # values shift backwards.
+  #
+  # Note: This will shift dates of the birthdate and
+  # dates on the data elements that exist on the patient.
+  def shift_dates(seconds)
```

Comment:
why was this added and where is it used? seems potentially dangerous, esp w setting the offset.

Reply:
Cypress had this functionality before, and so did we (ours was a rake task though). I agreed with the need for this from Cypress, and because I knew we needed similar functionality at certain points in Bonnie's history (yet implemented that same functionality separately on our own), I thought it made sense having this here.

What do you find dangerous about the offset? It basically just makes sure whatever values you are providing will be in UTC. See the added tests (that work in our timezone as well as Travis' time zone).

Reply:
adam and i had a discussion about this. @mayerm94 please remove `new_offset(0)` and see if tests behave as expected.

Reply:
It is done for all elements `element.shift_dates(seconds) `

Reply:
See line 34, this method calls `shift_dates` on all `data_elements` on this patient; see `shift_dates` in `data_element.rb`.

Reply:
Right, but the .new_offset(0) is not being called in the data_elements.shift_dates